hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/ConfigurationException.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.mesos.hdfs.config;
+
+/**
+ * Indicates an exception or poor request for configuration.
+ */
+public class ConfigurationException extends RuntimeException {
+
```

Comment:
We should also have a constructor with both a message and cause as params. I don't think we should even have the empty constructor.